sale_untaxed_amount_to_invoice_backport/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module Compute the untaxed amount already invoiced from the sale order line, taking the refund attached
```

Comment:
This module backports two fields from Odoo 12:
- Untaxed amount to invoice
- Untaxed invoices amoutn